javaparser-core/src/main/java/com/github/javaparser/ast/expr/StringLiteralExpr.java

```diff
@@ -78,6 +78,10 @@ public boolean remove(Node node) {
         return super.remove(node);
     }
 
+    public String asString() {
+        return value;
+    }
```

Comment:
Should it? What I see is that what takes two constructors for `CharLiteralExpr` should take two constructors for `StringLiteralExpr` too, but we have a typing problem now:
`CharLiteral(String) --> StringLiteral(String) // literal source code`
`CharLiteral(char) --> StringLiteral(String???) // translates special characters into source code escapes`

I don't have any good solutions though. Add an `isLiteralSourceCode` parameter? Maybe deprecate all the string constructors and use escaping as the default?